sentry_sdk/consts.py

```diff
@@ -504,6 +504,8 @@ def __init__(
         debug=None,  # type: Optional[bool]
         attach_stacktrace=False,  # type: bool
         ca_certs=None,  # type: Optional[str]
+        cert_file=None,  # type: Optional[str]
```

Comment:
whenever you add options here, please append to the end of the list since these are also positional.
No one should ideally use them positionally that way but we have to be safe.

Reply:
Thanks for your comment. `cert_file` and `key_file` are moved to the end of the arguments Now.